metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -405,6 +414,15 @@ class PowerBiDashboardSourceConfig(
         "Works only if configs `native_query_parsing`, `enable_advance_lineage_sql_construct` & `extract_lineage` are enabled.  "
         "Works for M-Query where native SQL is used for transformation.",
     )
+    extract_usage_stats: bool = pydantic.Field(
+        False,
+        description="Whether to ingest usage statistics for dashboards and reports.",
+    )
+    # TODO - stateful ingestion to autodetect usage stats interval
+    extract_usage_stats_for_interval: float = pydantic.Field(
```

Comment:
make it clear that this is in days

also should it be an int instead of a float?